### DISCUSSION TITLE
How to distinguish different acquisition protocols (PD, PDFS) in fastMRI knee single coil dataset?

### DISCUSSION THREAD

**Discussion by VINHYU** [facebookresearch/fastMRI#306] [Q&A] [answered]
I find the FastMRI paper mentions that there is an attribute named "acquisition" to indicate acquisition protocols in HDF5 files:
<img width="598" alt="Screen Shot 2023-05-22 at 19 21 00" src="https://github.com/facebookresearch/fastMRI/assets/54367728/0bbd06bb-3156-43c7-b991-5720c1156657">

However, I can't find this attribute in my downloaded knee single coil dataset. Or is there another way to get this information? Can someone help me?
<img width="1236" alt="Screen Shot 2023-05-22 at 19 25 47" src="https://github.com/facebookresearch/fastMRI/assets/54367728/c8e37d2d-3ac8-4f2d-bec8-110c911c1f85">

**Comment by mmuckley** [comment, marked as answer]
Hello @VINHYU, usually this information is in the `attr` field rather than the main data field.

**Reply by VINHYU** [reply]
It's true. I find it in hf.attrs field. Thank you very much!
